### ISSUE TITLE
A script creating voices fully automatically from voice datasets is needed

### ISSUE THREAD

**Issue by KOLANICH** [Olga-Yakovleva/RHVoice#162]
There are some free datasets targeted for creation of TTS and ASR systems, like Mozilla Common Voice.

I wonder if it is possible to create voices automatically from these datasets.

**Comment by rugk** [comment]
Yeah, wondered too, why you did not just use [the Common Voice project](https://commonvoice.mozilla.org/datasets). I don't really get what this issue is about, but it seems you do have different formats and need that to be "converted".
